app/poros/media_result.rb

```diff
@@ -0,0 +1,19 @@
+class MediaResult 
+  attr_reader :id,
+              :title,
+              :type,
+              :audience_score,
+              :poster
+
+  def initialize(media_result_data)
+    @id = media_result_data[:attributes][:id]
+    @title = media_result_data[:attributes][:title]
+    @type = media_result_data[:attributes][:type]
+    @audience_score = media_result_data[:attributes][:audience_score]
+    @poster = media_result_data[:attributes][:poster]
+  end
+
+  def formatted_type 
+    type.gsub("_", " ").titleize
+  end
```

Comment:
I have a generic media class in the branch I'm currently working on, is there any reason you think we would need this smaller media object? I know that some values will be nil because search results return less data but I think they can still be stored in the same object.